### DISCUSSION TITLE
请问有根据设备温度在启动风扇的插件吗？

### DISCUSSION THREAD

**Discussion by rianjskis** [immortalwrt/immortalwrt#1254] [General]
想固定设备温度到50度在启用风扇，设备是R2S，感谢

**Comment by m0eak** [comment]
https://github.com/JiaY-shi/fancontrol.git
可以试试这个，我gl-mt3000改了dts里的cooling-levels之后可以使用

**Reply by rianjskis** [reply]
感谢

**Reply by zyystudio** [reply]
> https://github.com/JiaY-shi/fancontrol.git 可以试试这个，我gl-mt3000改了dts里的cooling-levels之后可以使用

有编译好的么？现在都24.10.1immortalwrt还没有可直接使用版本

**Reply by m0eak** [reply]
> > https://github.com/JiaY-shi/fancontrol.git 可以试试这个，我gl-mt3000改了dts里的cooling-levels之后可以使用
> 
> 
> 
> 有编译好的么？现在都24.10.1immortalwrt还没有可直接使用版本

暂时没有
